### DISCUSSION TITLE
Which is better for finetuning `h2oai/h2ogpt-4096-llama2-7b-chat` or `h2oai/h2ogpt-4096-llama2-7b` on prepared question-answer pairs?

### DISCUSSION THREAD

**Discussion by sunilswain** [h2oai/h2o-llmstudio#758] [Q&A]
Hey
Let's assume I'm preparing a chatbot which would answer my queries based on my personal data. I've prepared around 4000 question-answer pairs for it. Which model should I choose as the backbone in my finetuning process?

Thanks

**Comment by psinger** [comment]
There is no clear-cut answer, you would need to try multiple options, but chat model might be the better choice.
